python/tvm/script/parse/parser.py

```diff
@@ -101,7 +101,7 @@ def eval_assign(
     ) -> Dict[str, Any]:
         var_values = eval_assign(self, target, source)
         for k, v in var_values.items():
-            var = bind_value(self, k, v)
+            var = bind_value(self, target, k, v)
```

Comment:
let's fix the type annotation for `bind_value` on line 100

Reply:
updated